cmd/agentbasedinstaller/register.go

```diff
@@ -66,7 +66,14 @@ func RegisterCluster(ctx context.Context, log *log.Logger, bmInventory *client.A
 	clusterParams := controllers.CreateClusterParams(&cd, &aci, pullSecret, releaseImageVersion, releaseImageCPUArch, nil)
 
 	if aci.Spec.Networking.NetworkType != "" {
-		clusterParams.NetworkType = &aci.Spec.Networking.NetworkType
+		if aci.Spec.Networking.NetworkType == "OpenshiftSDN" || aci.Spec.Networking.NetworkType == "OVNKubernetes" {
```

Comment:
Hi @karmab can you please provide some more details/context about this patch? Didn't find any mention in the PR description or OCPBUGS link.